protocols/api/src/main/java/org/apache/james/protocols/api/ProtocolConfigurationImpl.java

```diff
@@ -54,15 +54,12 @@ protected ProtocolConfigurationImpl(String softwareName) {
 
     @Override
     public String getHelloName() {
-        if (helloName == null) {
-            return DEFAULT_HELLO_NAME;
-        }
-        return helloName;
+        return DEFAULT_HELLO_NAME;
     }
     
     @Override
     public String getGreeting() {
-        return greeting;
+        return null;

```

Comment:
Why do we return null here?

Reply:
Because it is what the code do before the refactoring.
I guess it offers the possibility for subclasses to override this value.